third_party/ngraph/tbb.BUILD

```diff
@@ -0,0 +1,63 @@
+licenses(["notice"])  # 3-Clause BSD
```

Comment:
@thirupalanisamy would you please help to take a look at the license part of this file?
Thanks a lot.

Reply:
Looks good. BSD 3-Clause, MIT, and Apache 2.0 are already covered under "notice" only licenses.